src/python/dxpy/bindings/apollo/vizserver_client.py

```diff
@@ -0,0 +1,45 @@
+import dxpy
+
+class VizClient(object):
+    def __init__(self, url,project_id,record_id, error_handler=print) -> None:
+        self.url = url
+        self.project_id = project_id
+        self.record_id = record_id
+        self.error_handler = error_handler
+
+    def get_data(self,payload):
+        resource_val = "{}/data/3.0/{}/raw".format(self.url,self.record_id)
+        try:    
+            raw_response = dxpy.DXHTTPRequest(
+                resource=resource_val, data=payload, prepend_srv=False
+            )
+            if "error" in raw_response.keys():
+                if raw_response["error"]["type"] == "InvalidInput":
+                    err_message = "Insufficient permissions due to the project policy.\n" + raw_response["error"]["message"]
+                elif raw_response["error"]["type"] == "QueryBuilderError" and raw_response["error"]["details"] == "rsid exists in request filters without rsid entries in rsid_lookup_table.":
```

Comment:
Just checking - is it `["error"]["details"]` or  `["error"]["message"]`? I do not see `details` field mentioned in the API: https://github.com/dnanexus/hare/blob/master/docs/API-SpecificationV3.md#common-api-errors However, that does not mean it is not there.